flashlight/pkg/vision/CMakeLists.txt

```diff
@@ -2,7 +2,13 @@ cmake_minimum_required(VERSION 3.10)
 
 add_library(fl_pkg_vision "")
 
-target_link_libraries(fl_pkg_vision PUBLIC flashlight)
+target_link_libraries(fl_pkg_vision PUBLIC flashlight fl_lib_set)
+
+target_include_directories(
```

Comment:
nit: is this necessary for any of the pkgs? I would've thought this already works as per linking with the `flashlight` target. If it's necessary and I'm wrong, it can be left alone

Reply:
I was surprised by this as well. Let me double check and get back to you.